plugins/cumulus_frontend/web_client/js/views/body/ComputeResourcePanel.js

```diff
@@ -21,10 +21,32 @@ minerva.views.ComputeResourcePanel = minerva.views.Panel.extend({
     events: {
         'click .m-add-computeresource': 'addComputeResourceDialog',
         'click .m-cluster-details': 'viewClusterDetails',
+        'click .m-provision-cluster': 'provisionCluster',
         'click .m-terminate-cluster': 'terminateCluster',
         'click .m-remove-cluster': 'removeCluster'
     },
 
+    _clusterStatus: function (status) {
+        if (_.isUndefined(status)) {
+            return 'error';
+        } else if (Number.isInteger(status)) {
+            return {
+                0:   'creating',
+                10:  'created',
+                20:  'launching',
+                30:  'launched',
+                40:  'provisioning',
+                50:  'provisioned',
+                60:  'terminating',
+                70:  'terminated',
+                101: 'stopped',
+                102: 'running'
+            }[status];
+        } else {
```

Comment:
My preference would be to figure out why we're getting status codes instead of strings server-side and open a PR on cumulus